src/cmd/linuxkit/util.go

```diff
@@ -256,22 +256,27 @@ func NewPublishedPort(publish string) (PublishedPort, error) {
 }
 
 // CreateMetadataISO writes the provided meta data to an iso file in the given state directory
-func CreateMetadataISO(state, data string) ([]string, error) {
-	if data == "" {
+func CreateMetadataISO(state, data string, dataPath string) ([]string, error) {
+	var (
+		d   []byte
+		err error
+	)
```

Comment:
This is kind of verbose, using four lines to set two variables, and it still gets a shadowed use of `err` later, I think its only used in one `case` branch in the whole function...

Reply:
What do you suggest? Defining just `d` at the beginning of the `func` and `err` inside the `case` statement? Cannot just do `d, err := iotuil.ReadFile()` as it will shadow `d`.

Reply:
Yes, I think thats better, as `d` is used everywhere but `err` isnt.

Reply:
Done and pushed.